src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
Looks like rot13 by itself (no zip) is less overhead, only about 1.5%, and much more consistent.  I'll switch to that.